### PR TITLE
Slovakia VAT change from 2025/01 to 5% and 19% and 23%

### PR DESCRIPTION
**Pull request by ShaiMagal**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.2.x
| Description?      | Slovakia VAT change from 2025/01. Old rate: 10+20. New rate: 5+19+23, source for example: https://www.ui42.sk/blog/zmeny-sadzieb-dph-od-roku-2025-a-ich-dopad-na-e-shopy
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Install PrestaShop with SK package, you should get new rate 5+19+23
| UI Tests          | 
| Fixed issue or discussion?     | 
| Related PRs       | https://github.com/PrestaShop/LocalizationFiles/pull/51
| Sponsor company   | https://www.openservis.cz/

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `How to test` shouldn&#039;t be empty. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#how-to-test))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!

(Note: this is an automated message, but answering it will reach a real human)

**Review by Hlavtox — APPROVED** [review]

**Comment by Hlavtox** [comment]
DO NOT MERGE BEFORE 1.1.2025

**Comment by paulnoelcholot** [comment]
Hello @ShaiMagal 

I tested your PR and it's good for me!

Thanks!

**### Waiting 1.1.2025 to merge Thanks!**

image

**Comment by Hlavtox** [comment]
Ping @PrestaShop/committers for merge

**Review by marsaldev — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Comment by Caleydon** [comment]
Thanks for update. Shouldn't the Great Britain be removed from all localization XML files since it is no longer subject to the VAT reverse charge mechanism?

**Comment by Caleydon** [comment]
Addition: If the VAT rate changes, it should also be changed in the localization XML files of other affected countries where VAT reverse charge applies.

```
<tax id="30" name="DPH SK 20%" rate="20" auto-generated="1" from-eu-tax-group="virtual"/>
```
to
```
 <tax id="30" name="DPH SK 23%" rate="23" auto-generated="1" from-eu-tax-group="virtual"/>
```

**Comment by Caleydon** [comment]
If VAT changes in any country, PrestaShop could have a rule that it would release an updated version of PrestaShop on the day it comes into effect. So every new installation of PrestaShop would always create the current tax rules.

**Comment by ShaiMagal** [comment]
@Caleydon 
GB - You can make new PR for this, this PR is already merged and can't be changed, we are solving only Slovak VAT here. You are welcome to do a PR for GB. (PR can do everyone, we committers can approve/deny this)

Localization XML - It's already done via this PR in same date like thiS PR 37354 - https://github.com/PrestaShop/LocalizationFiles/pull/51

Vat change - it's not how prestashop release cycle works, update of PS have milestones. If there will be any inacurrate VAT, customer can change this easily at his own.

**Pull request merged**
